libs/utils/analytics.js

```diff
@@ -0,0 +1,19 @@
+export function decorateBlockAnalytics(blockEl) {
+  const lh = [];
+  const exclude = ['--', 'block'];
+  blockEl.classList.forEach((c) => {
+    if (!c.includes(exclude[0]) && c !== exclude[1]) lh.push(c);
+  });
```

Comment:
nit: inline exclude

```suggestion
  blockEl.classList.forEach((c) => {
    if (!c.includes('--') && c !== 'block') lh.push(c);
  });
```

Reply:
@chrischrischris  - Sorry, see my comment below. Good catch. This branch was out of date. I have updated this now if you could have another look

Reply:
@ryanmparrish we can remove the double dash check. With Helix 7, double dashes are removed.

Reply:
Donezo